docs/modules/integrate/pages/kafka-connector.adoc

```diff
@@ -103,6 +103,31 @@ properties.put("schema.registry.url", schemaRegistryUrl);
 Keep in mind that once the record deserialized, Jet still needs to know
 how to serialize/deserialize the record internally.
 
+== Embedded Avro Schema
+To use an embedded Avro schema for `GenericRecord`, you must provide the following as parameters for the Kafka producer and/or consumer:
+
+* The schema
+* A HazelcastKafkaAvroSerializer serialization class
+* A HazelcastKafkaAvroDeserializer deserialization class
+
+The parameters can be specified as shown in the following examples.
+
+Example consumer parameters:
+```java
+properties.put("key.deserializer", HazelcastKafkaAvroDeserializer.class);
+properties.put("value.deserializer", HazelcastKafkaAvroDeserializer.class);
+properties.put("keyAvroSchema", keyAvroSchema);
```

Comment:
is it obvious that `schema` is verbatim JSON representation of Avro schema? BTW, key and value may use different serializers, especially `key` might use some primitive type in order to facilitate predictable kafka partitioning

Reply:
about "key and value may use different serializers" I hoped that this would be obvious to the developer. Are you thinking about writing about this explicitly?

Reply:
I decided not to guess and added note about different serializers for key\value